app/Resources/views/app/main/baptism.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends 'app/layout.html.twig' %}
+
+{# translation default file for this view#}
+{% trans_default_domain 'baptism' %}
+
+{% block title %}
+{{ 'baptism.title' | trans({}) }} %}
```

Comment:
when you have a {% trans_default_domain %}, it means that |trans is enough, no need for the ({})